test/topology_custom/test_table_drop.py

```diff
@@ -0,0 +1,11 @@
+from test.pylib.manager_client import ManagerClient
+import pytest
+
+@pytest.mark.asyncio
+async def test_drop_table_during_streaming_receiver_side(manager: ManagerClient):
+    servers = [await manager.server_add(config={
+        'error_injections_at_startup': ['stream_mutation_fragments_table_dropped'],
+        'enable_repair_based_node_ops': False,
```

Comment:
I don't understand there's only a test that disables RBNO.

All those SCT reports are using the defaults, I think, and the default is to enable RBNO.

So we definitely should have a regression test for RBNO case as well. Preferably both